tests/api/api.spec.ts

```diff
@@ -556,6 +556,28 @@ describe("@dataform/api", () => {
           );
         });
     });
+    it("bigquery example with input backticks", () => {
+      return query
```

Comment:
nit: can we refactor these tests to use await/async please?

Reply:
Sure, I think I've done this right

Reply:
But I have been wrong before